src/main/osd/osd_warnings.c

```diff
@@ -76,6 +76,36 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
     static armingDisableFlags_e armingDisabledDisplayFlag = 0;
 
     warningText[0] = '\0';
+
+#ifdef USE_GPS_RESCUE
+    if (osdWarnGetState(OSD_WARNING_GPS_RESCUE_UNAVAILABLE)
+        && ARMING_FLAG(ARMED)
+        && gpsRescueIsConfigured()) {
+
+        statistic_t *stats = osdGetStats();
+        if (cmpTimeUs(stats->armed_time, OSD_GPS_RESCUE_DISABLED_WARNING_DURATION_US) < 0) {
+            tfp_sprintf(warningText, "RESCUE N/A");
+            // tell the user that it won't be available if requested
+            // but remove the message after the warning duration timeout (10s)
+            *displayAttr = DISPLAYPORT_SEVERITY_WARNING;
+            *blinking = true;
+            return;
+        }
+    }
+
+    if (osdWarnGetState(OSD_WARNING_GPS_RESCUE_FAILING)
+      && ARMING_FLAG(ARMED)
+      && FLIGHT_MODE(GPS_RESCUE_MODE)
+      && !gpsRescueIsHeadingOK()) {
```

Comment:
```suggestion
      && !gpsRescueIsOK()) {
```

Reply:
Thanks - I had to think a bit more about these.  I think they are correct now.  
There are two checks that are active immediately after arming:
- whether the rescue can work later on, i.e. if the hardware is not available, or no home point on arming.  This shows `RESCUE N/A` for 5s then goes away to stop annoying pilots who are taking off without a home point
- whether the IMU is oriented, subordinate to the previous message, which shows `HEADING N/A` until it is oriented
These two checks don't need precedence over the RXLOSS warning since they are cleared long before initiating a GPS RESCUE.
The other check should show `RESCUE FAIL` when a sanity check fails; these happen only once a Rescue has started while I the Rescue flight mode.